packages/api/src/__mocks__/env.ts

```diff
@@ -0,0 +1,7 @@
+export const env = {
+nodeEnv_Test : process.env.NODE_ENV,
+NODE_ENVP_TEST :"development",
+PORT_TEST : "3000",
+GITHUB_ID_TEST : "mock-client-id",
+GITHUB_SECRET_TEST : "Secrete_secret",
+};
```

Comment:
This should match the `env` object, it's just overriding that, so they don't come from `process.env` and come from here instead.

```suggestion
export const env = {
  nodeEnv: process.env.NODE_ENV,
  port : "3000",
  githubId : "mock-client-id",
  githubSecretId : "Secrete_secret",
};
```

Reply:
Not sure if those keys are correct, but they should be the same as in `env.ts`